src/modules/markets/markets.service.ts

```diff
@@ -15,7 +15,7 @@ export class MarketsService {
     let spResults = await this.marketsRepository.query('CIS_USP_READ_MARKETS');
```

Comment:
we should wrap the entire content of this function in a try-catch statement. This so that it doesn't crash the server if something goes sideways

Reply:
Added try catch. User gets generic error message and all details are logged. Maybe global exception handler can be added later.